src/views/registration-js/attendees/fields/birth.php

```diff
@@ -11,8 +11,9 @@
  * @see     Tribe__Tickets_Plus__Meta__Field__Birth
```

Comment:
Every time we introduce changes to a template file we need to also document this with something like:
`@since VERSION Descriptive text explaining the changes`.

Also this template is missing the`@version`. Each time we make changes to a template, we need to set the `@version TBD`.

Reply:
Added. Thanks. https://github.com/moderntribe/event-tickets/pull/1676/commits/c5b33a5f81300426a1a5443f2a2a56b4b7a47c05